README.md

```diff
@@ -68,17 +68,19 @@ csaf-validator --csaf-version 2.1 --test-id 6.1.34 my-csaf-2-1-document.json
 ```
 
 
-## Test-Conformance to the Standard
+## Implementation status in regards to the Standard
 
 * :white_check_mark: Implemented
 * :o: Not applicable
 
 ### Mandatory Tests
 
-| Testspecification | 2.0 | 2.1 (experimental) |
+| Test specification | 2.0 | 2.1 (experimental) |
 | --- | --- | --- |
 | 6.1.1 | :white_check_mark:  | |
```

Comment:
Checks are automatically implemented for both 2.0 and 2.1

Reply:
We can't generalize this, as there are different testcases for both versions.